.github/workflows/rector.yaml

```diff
@@ -0,0 +1,15 @@
+name: Code Analysis
```

Comment:
```suggestion
name: Rector
```

Reply:
Thanks.

I'm trying to figure out, why the workflow is not triggered bellow :/ any ideas?

Reply:
I tried in the past, it apply rectify, but after rectify, it stop, see:

https://github.com/rectorphp/rector-symfony/pull/121

I added unused private property:

https://github.com/rectorphp/rector-symfony/pull/121/commits/0206ad61d0af60b01f0101e5f988fb42a9f5bf38

It then run rectify:

https://github.com/rectorphp/rector-symfony/pull/121/commits/951fec105425bf17f4452e5402db25cf3f74818a

then stop as pending:

![Screen Shot 2022-03-08 at 17 07 36](https://user-images.githubusercontent.com/459648/157214867-fcf3ae3e-cd12-4dc2-b990-393c8bc97fb1.png)